docs/architecture/architecture.tex

```diff
@@ -80,16 +87,24 @@ \subsection{Subsystem decomposition}
 	\label{fig:modules}
 \end{figure}
 
-The input module should be able to read the data. With that data a dataModel
-can be constructed. \\
-The Analysis Specification module processes the script that defines how the
-data should be analyzed. The user provides this script. This module translate
-the script in operations that can be performed by the Analysis Module.\\
-The Analysis Module perform the analyses over the dataModel. The output of this
-module is the result of the analysis, this is a dataModel.\\
-The Visualization module creates a certain visual representation of a
-dataModel. For example it can create a box-plot of the creatine levels.
+\begin{description}
```

Comment:
beetje afhankelijk wat er op mijn bericht in telegram wordt geantwoord. Maar ik zou hier neer zetten dat momenteel de modules zo zijn, maar dat deze modules het model moeten gaan vormen en als het inderdaad klopte wat ik in telegram zette dan denk ik niet dat je de view en controller modules specifiek hoeft te noemen, dat doe je namelijk al in het stuk over MVC, hier wil je dan uitleggen hoe die modules zijn opgebouwd (en alleen bij model is wat nuttig te vertellen). 

Maar als je dit aanpast vertel dan hoe het nu zit en (mag in een zin) hoe het zou moeten worden, want voor hoe het nu is klopt het wel redelijk